### PR TITLE
PW-2477: added support for /orders and /orders/cancel, added order fi…

### PR DESCRIPTION
**Pull request by Aleffio**
…eld for /payments response

**Description**
added support for
/orders
/orders/cancel

orders field response in /payments

### CODE REVIEW

File: src/test/java/com/adyen/CheckoutTest.java [hunk 1]
```diff
@@ -1392,6 +1437,39 @@ protected CreatePaymentLinkRequest createPaymentLinkRequest() {
 
         return createPaymentLinkRequest;
     }
+
+    protected CheckoutCreateOrderRequest checkoutCreateOrderRequest() {
```
Comment (rkewlani) [inline src/test/java/com/adyen/CheckoutTest.java:1441]: `protected CheckoutCreateOrderRequest createCheckoutOrderRequest()`

Comment (rkewlani) [inline src/test/java/com/adyen/CheckoutTest.java:1441]:
```suggestion
protected CheckoutCreateOrderRequest createCheckoutCreateOrderRequest()
```

File: src/test/java/com/adyen/CheckoutTest.java [hunk 2]
```diff
@@ -1392,6 +1437,39 @@ protected CreatePaymentLinkRequest createPaymentLinkRequest() {
...
+        CheckoutOrder checkoutOrder = new CheckoutOrder();
+        checkoutOrder.setOrderData("Ab02b4c0!BQABAgBqxSuFhuXUF7IvIRvSw5bDPHN...");
+        checkoutOrder.setPspReference("8515930288670953");
+        paymentsRequest.setOrder(checkoutOrder);
+
+        paymentsRequest.setMerchantAccount("TestMerchant");
+        return paymentsRequest;
+    }
+
+    protected CheckoutCancelOrderRequest checkoutCancelOrderRequest() {
```
Comment (rkewlani) [inline src/test/java/com/adyen/CheckoutTest.java:1464]:
```suggestion
protected CheckoutCancelOrderRequest createCheckoutCancelOrderRequest()
```

File: src/test/java/com/adyen/CheckoutTest.java (hunk 1 above)
Comment (Aleffio) [reply src/test/java/com/adyen/CheckoutTest.java:1441]:
```suggestion
    protected CheckoutCreateOrderRequest createCheckoutCreateOrderRequest() {
```

File: src/test/java/com/adyen/CheckoutTest.java (hunk 2 above)
Comment (Aleffio) [reply src/test/java/com/adyen/CheckoutTest.java:1464]:
```suggestion
    protected CheckoutCancelOrderRequest createCheckoutCancelOrderRequest() {
```

**Pull request merged**